_components/form-templates.md

```diff
@@ -344,7 +392,52 @@ title: Forms Blocks
   </div>  
 </div>
 
-<h2 class="usa-heading">Contact Form</h2>
+<div class="preview" id="ssn-form">
+  <form>
+
+    <fieldset>
+      <legend>Social Security Number</legend>
+
+      <label for="ssn">SSN <span class="usa-additional_text">Required</span>
+      </label>
+      <input id="ssn" name="ssn" type="password" class="usa-ssn"
+          data-grouplength="3,2,4" data-delimiter="-" data-politespace
+          required pattern="^d{8}$">
+      <p class="usa-form-note">
+        <a title="Show password" href="javascript:void(0)"
+            class="usa-show_ssn"
+            aria-controls="ssn">
+          Show SSN</a>
+      </p>
+
+    </fieldset>
+
+  </form>
+</div>
+
+<div class="usa-grid">
```

Comment:
These are using the old documentation scheme, and I had already updated these.

Am I correctly looking at the code where you fixed the merge conflicts?